internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -456,7 +456,7 @@ func resourceCosmosDbAccount() *pluginsdk.Resource {
 				},
 			},
 
-			"identity": commonschema.SystemAssignedIdentityOptional(),
+			"identity": commonschema.SystemAssignedUserAssignedIdentityOptional(),
```

Comment:
can we add [an acceptance test](https://github.com/hashicorp/terraform-provider-azurerm/blob/35d55df522d8e79ec8c59eca7ce9d67d1a045ab2/internal/services/cosmos/cosmosdb_account_resource_test.go#L781-L810) covering adding a User Assigned Identity?

Reply:
I've updated `TestAccCosmosDBAccount_identity` to provision a test managed identity and then to set the type from `SystemAssigned` to `SystemAssigned, UserAssigned` so that SystemAssigned and UserAssigned identities are associated with the resource in a single test